imls-backend/db/migrations/20220923222643_create_search_lib_fscs.sql

```diff
@@ -0,0 +1,18 @@
+-- migrate:up
+CREATE OR REPLACE FUNCTION api.lib_search_fscs(
+	_fscs_id text)
+    RETURNS JSON
+    LANGUAGE 'sql'
+    COST 100
+AS $$
+SELECT row_to_json(X) FROM
+(SELECT *, CONCAT(fscskey,'-',TO_CHAR(fscs_seq, 'fm000')) AS fscs_id 
```

Comment:
Just a quick note, would have been nice to have a comment here re: template pattern modifier `fm000` as I had to look up what this does.

Reply:
I will add to the documentation for the future, thank you.